pkgs/top-level/all-packages.nix

```diff
@@ -13828,6 +13828,7 @@ let
 
   weechat = callPackage ../applications/networking/irc/weechat {
     inherit (darwin) libobjc;
+    inherit (pythonPackages) python pycrypto pync;
```

Comment:
please add back pythonPackages as a input and then check for presence of pycrypto and pync. that way there is not way that they can be of different python version. this is especially important when you override original derivation.